nautobot_chatops/dispatchers/slack.py

```diff
@@ -297,7 +297,8 @@ def get_prompt_from_menu_choices(self, choices, offset=0):
             try:
                 # Since we are showing "Next..." at the end, this isn't required to show to users anymore
                 self.send_warning(
-                    f"More than {self.slack_menu_limit} options are available. Slack limits us to only displaying {self.slack_menu_limit} options at a time."
+                    f"More than {self.slack_menu_limit} options are available. \
+                     Slack limits us to only displaying {self.slack_menu_limit} options at a time."
```

Comment:
I think the below would work as well and would avoid using backslash-newline?

```suggestion
                    f"More than {self.slack_menu_limit} options are available. "
                    f"Slack limits us to only displaying {self.slack_menu_limit} options at a time."
```

Reply:
Makes sense, I will incorporate these as well.

Reply:
This is done